ua-snap/data-api
# Modifies documentation to use academic citation format

Simplifies the Data Sources section a bit further, reducing the number of links that we may need to maintain & simply referring out to certain external works in general instead of specific files (more brittle).

Also some small changes elsewhere in the documentation.

To test this:

 - Visit each documentation endpoint.  Each should have a citation-style reference out to the relevant academic work(s) [[ some won't -- the endpoints we are removing, or ones we don't have final academic references for yet -- Landfast Ice, Design Indices, Beetles, Glaciers, Mean Annual Ground Temp ]]
 - Check that the table isn't mangled and the layout is "reasonable" (we don't super-care about looks here)
